build/Package.targets

```diff
@@ -7,10 +7,11 @@
   <Import Project="$(MSBuildThisFileDirectory)/package/Installer.MSI.targets" />
   <Import Project="$(MSBuildThisFileDirectory)/package/Installer.PKG.targets" />
   <Import Project="$(MSBuildThisFileDirectory)/package/Installer.RPM.targets" />
+  <Import Project="$(MSBuildThisFileDirectory)/package/CopySandBoxPackageOut.targets" />
 
   <Target Name="GenerateInstallers"
           Condition=" '$(SkipBuildingInstallers)' != 'true' "
-          DependsOnTargets="Init;Layout;GeneratePkgs;GenerateDebs;GenerateMsis;GenerateRpms" />
+          DependsOnTargets="Prepare;Layout;GeneratePkgs;GenerateDebs;GenerateMsis;GenerateRpms" />
```

Comment:
Why the change from Init to Prepare?

Reply:
"DownloadHostAndSharedFxArtifacts" is actually required for Package. And init does not have it

```xml
<Target Name="Prepare"
          DependsOnTargets="Init;DownloadHostAndSharedFxArtifacts;RestoreSrcPackages;RestoreToolsPackages;IsolateArtifactsFromDirectoryBuildFiles;CreatePropsForNextStage" />

  <Target Name="Init"
          DependsOnTargets="PrintBuildInfo;
                            SetTelemetryProfile;
                            BuildDotnetCliBuildFramework;
                            CheckPrereqs;">
```